tests/integration/ceph_mgr_test.go

```diff
@@ -121,20 +121,60 @@ func (suite *CephMgrSuite) execute(command []string) (error, string) {
 	return suite.cluster.installer.Execute("ceph", orchCommand, suite.namespace)
 }
 
+func (suite *CephMgrSuite) enableRookOrchestrator() (error) {
+	logger.Info("Enabling Rook orchestrator module: <ceph mgr module enable rook>")
+	err, output :=suite.cluster.installer.Execute("ceph", []string{"mgr", "module", "enable", "rook"}, suite.namespace)
```

Comment:
The build is failing due to whitespace on this line. A linter will help you catch this issue.

Reply:
Solved